EIPS/eip-4804.md

```diff
@@ -0,0 +1,65 @@
+---
+eip: 4804
+title: Exodus
+description: Exodus allows to create blockchain networks with accounts and transactions.
+editors: Antoine Toulmé (@atoulme), Chaals Nevile (@chaals)
+discussions-to: <https://entethalliance.org/contact>, For EEA members only (requires a patent commitment): <https://github.com/entethalliance/testing-wg/issues> 
+status: Draft
+type: Standards Track
+category (*only required for Standards Track):ERC
+created: 2021-11-09
+requires (*optional): See [normative references]()
```

Comment:
```suggestion
```
This should be a list of EIPs or (if there are no EIP dependencies) it should not exist.